app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -0,0 +1,80 @@
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.reject_by_default.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">The last day for training providers to make a decision on all applications for courses starting in September <%= @recruitment_cycle_year %>.</td>
+          </tr>
+        </tbody>
+      </table>
+    <% else %>
+      <h1 class="govuk-heading-l">The application process for courses starting in September <%= @recruitment_cycle_year %></h1>
+      <p class="govuk-body">For courses starting in September <%= @recruitment_cycle_year %>, you can start applying a year before they start in October <%= @recruitment_cycle_year - 1 %>.</p>
+      <p class="govuk-body">Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.</p>
+
+      <br>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Date and time</th>
+            <th scope="col" class="govuk-table__header">What happens</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.find_opens.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">Start <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses" class="govuk-link">finding postgraduate teacher training courses</a> you might want to apply to.</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.apply_opens.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">Start applying to courses. You can apply to 4 courses offered by 4 different training providers. Courses can fill up quickly, so you should apply as soon as you can.<br><br>You can withdraw an application at any time and apply to a different course if you want to.<br><br>Training providers will have 30 working days to look at your application and decide to offer you a place on a course. Holiday periods in England over December and March or April are not included in the 30 working days. Each course you apply to will show the date for when providers need to respond by.<br><br>If training providers do not respond in time, you'll be able to apply to another course while you wait for a response. We'll tell you if the provider has not responded in time.<br><br>You’ll get an email if you receive an offer from a training provider.</td>
```

Comment:
Feel free to ignore. Do we want to make use of the global constant here for application choices? Means we won't have to come back and edit this at a later date?

MAXIMUM_NUMBER_OF_COURSES = 4.
`apply-for-teacher-training/app/models/application_form.rb`